### PR TITLE
11 ms sample data

### PR DESCRIPTION
**Pull request by MayaS1111**
Created sample data for the following models:
- User
- Program
- Cohort
- Program Admin
- Cohort Member
- Match

I also added a 'running?' association to cohorts to see if the cohort is still running based on when the ending date is.

*If you get an error when running 'rails sample_data' just rerun it. The error is coming from off-role ratios; nothing for you to worry about 😊.

### CODE REVIEW

File: lib/tasks/dev.rake [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  people = Array.new(38) do
+    {
+      first_name: Faker::Name.first_name,
+      last_name: Faker::Name.last_name
+    }
+  end
+  people << { first_name: 'Alice', last_name: 'Smith' }
+  people << { first_name: 'Bob', last_name: 'James' }
+
+  image_name = %w[Buddy Mimi Abby Jack Lily Lucy Jasmine Bear Loki Dusty Maggie Milo Lucky
```
Comment (borvux) [inline lib/tasks/dev.rake:22]: Make comment for how this is used

File: lib/tasks/dev.rake [hunk 2]
```diff
@@ -0,0 +1,169 @@
+desc 'Fill the database tables with some sample data'
+task({ sample_data: :environment }) do
+  starting = Time.now
+  p "Creating sample data..."
+
+  Match.delete_all
```
Comment (borvux) [inline lib/tasks/dev.rake:6]: could potentially put all of this into 
```ruby
if Rails.env.development?
  Match.delete_all
  CohortMember.delete_all
  ProgramAdmin.delete_all
  Cohort.delete_all
  Program.delete_all
  User.delete_all
end
```
could helps prevent unintended data loss in production

**Review by borvux — APPROVED** [review]
looks good to me, could turn larger code into smaller methods but refactoring could be done later for it

**Review by rezahusain — APPROVED** [review]
Could probably add a couple comments just to clarify what some of the lines are doing but overall really solid!

**Pull request merged**